packages/ui/src/common/Accordion/Accordion.tsx

```diff
@@ -58,13 +59,17 @@ const accordionStyle = css`
  * @param accordionItems - Array of accordion items to render
  * @param collapseAll - Controls initial state and dynamic collapse/expand of all items. true = collapsed, false = expanded
```

Comment:
Classic case of "added a param but didn't update the comments". It would be helpful to know what this `selectedIndex` does cause I can't guess from context.